crates/lib/kajiya/src/renderers/csgi.rs

```diff
@@ -182,6 +182,7 @@ impl CsgiRenderer {
         rg: &mut rg::TemporalRenderGraph,
         sky_cube: &rg::Handle<Image>,
         bindless_descriptor_set: vk::DescriptorSet,
+        #[cfg(feature = "ray-tracing")]
         tlas: &rg::Handle<RayTracingAcceleration>,
```

Comment:
I've got these `#[cfg(feature = "ray-tracing")]`s everywhere instead of just disabling these passes outright. This is messy and probably the wrong way to do things.